server/bin/test_data.json

```diff
@@ -1088,10 +1089,7 @@
                 "virt_limit" : "unlimited",
                 "host_limited": "true"
             },
-            "derived_product_id" : "awesomeos-server-basic-vdc",
-            "derived_provided_products" : [
-                "37080", "37060"
-            ]
+            "derived_product_id" : "awesomeos-server-basic-vdc"
```

Comment:
Having a product provide itself as the derived product does not seem correct. I believe this test product was for one that provides no content itself (it would have no provided products) but is linked to a derived product that would provide the content.

Reply:
I was going to say the same, but then I realized it is correct. the basic-dc is the main product (which has no provided products), which points to the basic-vcd (with this field), which then does have provided products.